esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
+{% autoescape off %}
+{{ msgbdy }}
+{% endautoescape %}
```

Comment:
If we're allowing both HTML and plain text, we probably actually want to have two templates; the plain-text one can just include the URL.

What do you mean by phone-compatibility code?